Extension/src/main.ts

```diff
@@ -279,6 +284,12 @@ async function postInstall(info: PlatformInformation): Promise<void> {
 }
 
 async function finalizeExtensionActivation(): Promise<void> {
+    if (vscode.workspace.getConfiguration("C_Cpp", null).get<string>("intelliSenseEngine") === "Disabled") {
```

Comment:
There doesn't seem to be a way to re-enable the language server after it's been disabled.  It also appears that disabling the language server after it has already been loaded will have no effect.

Reply:
Yeah, that was "by design" :)...my assumption was that users who wanted to use this setting wouldn't mind having to Reload Window for it to work.

Reply:
I added the reload prompt.